packages/create-svelte/cli/index.js

```diff
@@ -77,9 +77,14 @@ async function main() {
 	await prompt_modifications(target);
 
 	console.log(
-		'\nWant to add other parts to your code base? ' +
-			'Visit https://github.com/svelte-add/svelte-adders, a community project of commands ' +
-			'to add particular functionality to Svelte projects\n'
+		`\n✧ Want to add other parts to your code base or add support for ${bold(
+			cyan('CSS preprocessors')
+		)}?`
+	);
+	console.log(
+		`\nVisit ${bold(
+			cyan('https://github.com/svelte-add/svelte-adders')
+		)} and unleash the possibilities with a community project of commands to add particular functionalities to Svelte projects.\n`
```

Comment:
This reads a little too non-neutral for a CLI setup for me 😄 not sure what others think about that

Reply:
Yes, I agree. I'd like to see how the `svelte-adders` project develops before we make it an official recommendation.

Reply:
It's already in the CLI and it's fortunately very easy to miss because I'm not interested in having a large amount of attention (yet).

Reply:
It's a community-made and also really buggy code modification tool, so everywhere it's linked to should be removed or replaced with a link to the [official integrations repo](https://github.com/sveltejs/integrations).

Reply:
@dummdidumm @babichjacob Oh I see, it makes sense :) What do you recommend then? Replace the svelte-adders link with the official integrations repo one?

Reply:
Sure, that's good.

Reply:
Done :) 

<img width="764" alt="Screen Shot 2021-04-12 at 13 36 35" src="https://user-images.githubusercontent.com/9087886/114429761-1bdf0d80-9b94-11eb-827e-e17998813d02.png">